crates/astria-conductor/src/sequencer/tests.rs

```diff
@@ -0,0 +1,110 @@
+use core::time::Duration;
+use std::sync::OnceLock;
+
+use serde_json::json;
+use tendermint_rpc::HttpClient;
+use wiremock::MockServer;
+
+use crate::{
+    config::CommitLevel,
+    executor,
+    metrics::Metrics,
+    sequencer,
+    sequencer::CancellationToken,
+};
+
+async fn mount_genesis(mock_http: &MockServer, sequencer_chain_id: &str) {
```

Comment:
can you use the existing `helpers::mount_genesis` function instead of this? since this is only  used in the chain-id mismatch test, you can just set the `sequencer_chain_id` in the builder to a different value than the default set in `helpers::mount_genesis`.

Reply:
I might be mistaken, but I believe `helpers::mount_genesis` is only available to `tests/blackbox`. I created the test file within `src/sequencer` so that the functionality of the chain ID check could be tested without doing all the other necessary mounts/calls for the full conductor. Would you prefer to move this to the `tests/blackbox` folder and perform the mounts so that it can utilize `helpers::mount_genesis`?